### PR TITLE
Lesson-1-8 Submission with some errors

### PR DESCRIPTION
**Pull request by Tugeeta**
Some syntax errors are there, tried, but to no avail. Help please!

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -2,38 +2,66 @@ import  React, { useEffect, useState } from 'react';
...
+            const options = {
+              method: 'GET',
+              headers: {
+                Authorization: `Bearer ${process.env.REACT_APP_AIRTABLE_API_KEY}`,
+            }},
+          
+            
+     url = (`https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/${process.env.REACT_APP_TABLE_NAME}\`,{
+          
+            });
```
Comment (XeniyaDob) [inline src/App.js:32]: Declaration of variable url
```const url = `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/${process.env.REACT_APP_TABLE_NAME}/```
Remove not needed `,{ });` from line 30-32

File: src/App.js [hunk 2]
```diff
@@ -2,38 +2,66 @@ import  React, { useEffect, useState } from 'react';
...
 
-        const [isLoading, setIsLoading] = useState(true);
+  const [isLoading, setIsLoading] = useState(true);
   
+    async function fetchData (
+   
+        
+        useEffect = () => {
+         fetchData () 
+        }, []);
```
Comment (XeniyaDob) [inline src/App.js:20]: This useEffect should be after your fetch data function

File: src/App.js [hunk 3]
```diff
@@ -2,38 +2,66 @@ import  React, { useEffect, useState } from 'react';
...
-    const loadTodoList = new Promise((resolve, reject) => {
-    setTimeout(() => {
-      resolve({ data: { todoList: todoList  } });
-    }, 2000);
-  
-  });
+             if (!response.ok) {
+              const message = ('Error occurred: ${response.status}');
+              throw new Error(message);               
+            };
```
Comment (XeniyaDob) [inline src/App.js:43]: This conditional statement should be inside `try` block

File: src/App.js [hunk 4]
```diff
@@ -2,38 +2,66 @@ import  React, { useEffect, useState } from 'react';
...
+          
+            
+     url = (`https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/${process.env.REACT_APP_TABLE_NAME}\`,{
+          
+            });
+             try {    
+          const response = await fetch (url, options);
+               console.log(response);
+               const data = await response.json();
+               console.log("API Data:", data);
```
Comment (XeniyaDob) [inline src/App.js:37]: Below the `if` block, declare a variable, data

File: src/App.js [hunk 5]
```diff
@@ -2,38 +2,66 @@ import  React, { useEffect, useState } from 'react';
...
+              Id: records.Id,
+              Title: records.fields.Title
+              
+           }
+             return (todos)      
+            
 
-  loadTodoList
-      .then((result) => {
+          )};
```
Comment (XeniyaDob) [inline src/App.js:55]: In homework 1.8, use this example.
```
const todos = todosFromAPI.records.map((todo) => {

              const newTodo =  {
                  id: todo.id,
                  title: todo.fields.title
              }

              return newTodo

          })
```

**Review by XeniyaDob — COMMENTED** [review]
Hi Geeta,

Thank you for working hard on your project. There are things you need to fix before working on the next assignment. 
I added comments to your last commit. Please try to debug and make the app work.  Debugging is an essential skill in the learning process, and it will greatly contribute to your growth as a developer. I believe in your ability to identify and resolve any issues that may arise. If you are struggling with debugging, I encourage you to schedule a one-on-one mentor session and review the code with any available mentor.
 Thank you for your dedication and effort in honing your coding skills.

**Comment by Tugeeta** [comment]
Hi 
Thank you for reviewing my work.  I appreciated it.  I will do what you have suggested.

**Pull request merged**